k4Interface/include/k4Interface/INoiseConstTool.h

```diff
@@ -24,7 +24,7 @@
 
 /** @class INoiseConstTool
  *
- *  Abstract interface to get calorimeter noise per cell tool
+ *  Abstract interface to get calorimeter noise constants (RMS and offset) per cell tool
```

Comment:
Can the naming be also harmonized? I mean to use "noise values" everywhere or "noise constants" :)